bindata/manifests/sriov-config-service/openshift/sriov-config-service.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
```

Comment:
I am a bit concerned about the impact of a cluster with a mix of workers that have sriov and don't have sriov. This machine config applies to all workers and uses the default worker MCP pool that comes with openshift. This might trigger the cluster to go down for a very long period causing all workers to reboot. I see this as per design with machine config...

Reply:
Would it be better to use labeling of nodes that want sriov-network-operator in systemd? But then we cannot use the default worker MCP pool which does complicate things.

Reply:
there is a way to select different pool

https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/365/files#diff-4be06e26c778a4f59b86cf746f89bbe18b78828a97ab43ac0c39c97b1d6532d6R365

Reply:
Got it thanks!